### PR TITLE
Vpn examples

### PR DESCRIPTION
**Pull request by bridgetdMSFT**
This checklist is used to make sure that common issues in a pull request are addressed. This will expedite the process of getting your pull request merged and avoid extra work on your part to fix issues discovered during the review process.

### PR information
- [ ] The title of the PR is clear and informative.
- [ ] There are a small number of commits, each of which have an informative message. This means that previously merged commits do not appear in the history of the PR. For information on cleaning up the commits in your pull request, [see this page](https://github.com/Azure/azure-powershell/blob/master/documentation/development-docs/cleaning-up-commits.md).
- [ ] Except for special cases involving multiple contributors, the PR is started from a fork of the main repository, not a branch.
- [ ] If applicable, the PR references the bug/issue that it fixes.
- [ ] Swagger files are correctly named (e.g. the `api-version` in the path should match the `api-version` in the spec).

### Quality of Swagger
- [ ] I have read the [contribution guidelines](https://github.com/Azure/azure-rest-api-specs/blob/master/.github/CONTRIBUTING.md).
- [ ] My spec meets the review criteria:
  - [ ] The spec conforms to the [Swagger 2.0 specification](https://github.com/OAI/OpenAPI-Specification/blob/master/versions/2.0.md).
  - [ ] The spec follows the guidelines described in the [Swagger checklist](https://github.com/Azure/azure-rest-api-specs/blob/master/documentation/swagger-checklist.md) document.
  - [ ] [Validation tools](https://github.com/Azure/azure-rest-api-specs/blob/master/documentation/swagger-checklist.md#validation-tools-for-swagger-checklist) were run on swagger spec(s) and have all been fixed in this PR.

### CODE REVIEW

**Comment by dsgouda** [comment]
@bridgetdMSFT please fix issues with `VirtualNetworkGatewaySupportedVpnDevice.json` doesn't look like a valid json file. 
Moreover, please verify what your service is producing/consuming, in the examples provided you are returning an xml in the body whereas the specification says the services consumes/produces `application/json` 
Please address the produces/consumes for the related operation

File: specification/network/resource-manager/Microsoft.Network/stable/2018-01-01/virtualNetworkGateway.json [hunk 1]
```diff
@@ -447,6 +465,9 @@
             "description": "Accepted and the operation will complete asynchronously."
           }
         },
+        "x-ms-examples": {
+          "Generate Virtual Network Gateway VPN Profile": { "$ref": "./examples/VirtualNetworkGatewayGenerateVpnProfile.json" }
```
Comment (dsgouda) [inline specification/network/resource-manager/Microsoft.Network/stable/2018-01-01/virtualNetworkGateway.json]: By convention, json keys are named without spaces. Please fix in other instances too

**Review by dsgouda — CHANGES_REQUESTED** [review]
Please address model validation errors reported [here](https://api.travis-ci.org/v3/job/356066171/log.txt)

File: specification/network/resource-manager/Microsoft.Network/stable/2018-01-01/virtualNetworkGateway.json (hunk 1 above)
Comment (bridgetdMSFT) [reply specification/network/resource-manager/Microsoft.Network/stable/2018-01-01/virtualNetworkGateway.json]: Removed spaces from example keys. Thanks for pointing that out.

**Comment by dsgouda** [comment]
@bridgetdMSFT  there are some validation errors with the examples reported [here](https://api.travis-ci.org/v3/job/356066171/log.txt) please take a look

**Comment by dsgouda** [comment]
@bridgetdMSFT Please reopen the PR against March branch or against master if it is just for examples. Please sync up with @MikhailTryakhov

**Pull request closed without merging**